api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -4896,3 +4931,89 @@ message OktaAssignmentResult {
     (gogoproto.jsontag) = ""
   ];
 }
+
+// AccessListResource is emitted when an access list is created, updated, or deleted.
+message AccessListResource {
+  // metadata is common event metadata
+  Metadata metadata = 1 [
+    (gogoproto.nullable) = false,
+    (gogoproto.embed) = true,
+    (gogoproto.jsontag) = ""
+  ];
+
+  // resource is common resource event metadata
+  ResourceMetadata resource = 2 [
+    (gogoproto.nullable) = false,
+    (gogoproto.embed) = true,
+    (gogoproto.jsontag) = ""
+  ];
+
+  // status indicates whether the resource operation was successful.
+  Status status = 3 [
+    (gogoproto.nullable) = false,
+    (gogoproto.embed) = true,
+    (gogoproto.jsontag) = ""
+  ];
+}
+
+// AccessListMemberResource is emitted when an access list member is created, updated, or deleted.
+message AccessListMemberResource {
```

Comment:
This is an odd name for an event, don't you think? 

- is an "access list member" even a resource, or is the access list itself the resource
- since events are typically recording what actions have occurred, can we think of a name that better reflects an action (verb) as opposed to a noun?

Reply:
AccessListMember is its own resource.

I could potentially suffix these with `Event`. Thoughts?